java/README.md

```diff
@@ -184,6 +244,9 @@ public class Main {
 }
 ```
 
+### Accessing tests
+You can access [unit tests](./java/client/src/test/java/glide/api) and [integration tests](./java/integTest/src/test/java/glide) developed for the Valkey commands here.
```

Comment:
Why do that for a user? You can tell that they could be used as examples. UT are useless though.

Reply:
Alexey suggested this because our documentation is lacking, apparently

Reply:
I don't mind having that, just a bit more details.

Reply:
Suggesting edit: 

For more examples, you can refer to the test folder  [unit tests](./java/client/src/test/java/glide/api) and [integration tests](./java/integTest/src/test/java/glide)